docker-compose.dev.yml

```diff
@@ -12,26 +13,9 @@ services:
       - ENVIRONMENT=development
     restart: always
     volumes:
-      - resin-data:/data
+      - anthias-data:/data
       - ./:/usr/src/app/
 
-# We might bring this in in the future
-#  anthias-viewer:
-#    build:
-#      context: .
-#      dockerfile: docker/Dockerfile.viewer
-#    depends_on:
-#      - anthias-server
-#    environment:
-#      - HOME=/data
```

Comment:
Don't leave behind commented out code. We have git history for that.

Reply:
Got it. This will be removed in this PR.